Source/Windows/ARM64EC/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ target_link_libraries(arm64ecfex
   ntdll_ex
 )
 
-target_link_options(arm64ecfex PRIVATE -static -nostdlib -nostartfiles -nodefaultlibs -lc++ -lc++abi -lunwind -lclang_rt.builtins-arm64ec)
+target_link_options(arm64ecfex PRIVATE -static -nostdlib -nostartfiles -nodefaultlibs -lc++ -lc++abi -lunwind ${libgcc_path})
```

Comment:
Should this be moved to a separate `target_link_libraries` line? On my local system, clang and gcc both return a plain filename without `-l` prefix; not sure if mingw behaves differently though.